jobs/JEVS_CAM_PLOTS

```diff
@@ -107,7 +110,7 @@ echo $COMPATH
 # Execute the script.
 #######################################################################
 
-if [ $MODELNAME = firewxnest ]; then
+if [ $MODELNAME = nam_firewxnest ]; then
```

Comment:
Marcel's PR changed this to `if [ $MODELNAME = firewxnest ]; then`, please restore.